### ISSUE TITLE
Tagging errors for "i.e."

### ISSUE THREAD

**Issue by muchang** [UniversalDependencies/UD_English-EWT#219]
Hi, some of "i.e."s in EWT are tagged as ADV, while some of them are tagged as ADJ. It seems to be ADV, thus, I post the ADJ cases here. 

```
# sent_id = weblog-blogspot.com_dakbangla_20050311135387_ENG_20050311_135387-0208
# text = Greendale likely refers to green river valley -- i.e., Cairo's Egyptian Islamic Jihad or the Islamic Group.
1	Greendale	Greendale	PROPN	NNP	Number=Sing	3	nsubj	3:nsubj	_
2	likely	likely	ADV	RB	_	3	advmod	3:advmod	_
3	refers	refer	VERB	VBZ	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	0:root	_
4	to	to	ADP	IN	_	7	case	7:case	_
5	green	green	ADJ	JJ	Degree=Pos	7	amod	7:amod	_
6	river	river	NOUN	NN	Number=Sing	7	compound	7:compound	_
7	valley	valley	NOUN	NN	Number=Sing	3	obl	3:obl:to	_
8	--	--	PUNCT	:	_	7	punct	7:punct	_
9	i.e.	i.e.	ADJ	FW	_	15	amod	15:amod	SpaceAfter=No
10	,	,	PUNCT	,	_	15	punct	15:punct	_
11-12	Cairo's	_	_	_	_	_	_	_	_
11	Cairo	Cairo	PROPN	NNP	Number=Sing	15	nmod:poss	15:nmod:poss	_
12	's	's	PART	POS	_	11	case	11:case	_
13	Egyptian	Egyptian	ADJ	JJ	Degree=Pos	15	amod	15:amod	_
14	Islamic	Islamic	ADJ	NNP	Degree=Pos	15	amod	15:amod	_
15	Jihad	Jihad	PROPN	NNP	Number=Sing	7	appos	7:appos	_
16	or	or	CCONJ	CC	_	19	cc	19:cc	_
17	the	the	DET	DT	Definite=Def|PronType=Art	19	det	19:det	_
18	Islamic	Islamic	ADJ	NNP	Degree=Pos	19	amod	19:amod	_
19	Group	Group	PROPN	NNP	Number=Sing	15	conj	7:appos|15:conj:or	SpaceAfter=No
20	.	.	PUNCT	.	_	3	punct	3:punct	_
```

```
# sent_id = email-enronsent16_01-0040
# text = For banks incorporated in a non-approved jurisdiction, it may be possible for them to trade out of a branch located in an approved jurisdiction (i.e. U.S. and U.K. branches).
1	For	for	ADP	IN	_	2	case	2:case	_
2	banks	bank	NOUN	NNS	Number=Plur	12	obl	12:obl:for	_
3	incorporated	incorporate	VERB	VBN	Tense=Past|VerbForm=Part	2	acl	2:acl	_
4	in	in	ADP	IN	_	7	case	7:case	_
5	a	a	DET	DT	Definite=Ind|PronType=Art	7	det	7:det	_
6	non-approved	non-approved	ADJ	JJ	Degree=Pos	7	amod	7:amod	_
7	jurisdiction	jurisdiction	NOUN	NN	Number=Sing	3	obl	3:obl:in	SpaceAfter=No
8	,	,	PUNCT	,	_	12	punct	12:punct	_
9	it	it	PRON	PRP	Case=Nom|Gender=Neut|Number=Sing|Person=3|PronType=Prs	12	expl	12:expl	_
10	may	may	AUX	MD	VerbForm=Fin	12	aux	12:aux	_
11	be	be	AUX	VB	VerbForm=Inf	12	cop	12:cop	_
12	possible	possible	ADJ	JJ	Degree=Pos	0	root	0:root	_
13	for	for	SCONJ	IN	_	16	mark	16:mark	_
14	them	they	PRON	PRP	Case=Acc|Number=Plur|Person=3|PronType=Prs	16	nsubj	16:nsubj	_
15	to	to	PART	TO	_	16	mark	16:mark	_
16	trade	trade	VERB	VB	VerbForm=Inf	12	csubj	12:csubj	_
17	out	out	ADP	IN	_	20	case	20:case	_
18	of	of	ADP	IN	_	20	case	20:case	_
19	a	a	DET	DT	Definite=Ind|PronType=Art	20	det	20:det	_
20	branch	branch	NOUN	NN	Number=Sing	16	obl	16:obl:of	_
21	located	locate	VERB	VBN	Tense=Past|VerbForm=Part	20	acl	20:acl	_
22	in	in	ADP	IN	_	25	case	25:case	_
23	an	a	DET	DT	Definite=Ind|PronType=Art	25	det	25:det	_
24	approved	approve	VERB	VBN	Tense=Past|VerbForm=Part	25	amod	25:amod	_
25	jurisdiction	jurisdiction	NOUN	NN	Number=Sing	21	obl	21:obl:in	_
26	(	(	PUNCT	-LRB-	_	31	punct	31:punct	SpaceAfter=No
27	i.e.	i.e.	ADJ	FW	_	31	amod	31:amod	_
28	U.S.	U.S.	PROPN	NNP	Number=Sing	31	compound	31:compound	_
29	and	and	CCONJ	CC	_	30	cc	30:cc	_
30	U.K.	U.K.	PROPN	NNP	Number=Sing	28	conj	28:conj:and|31:compound	_
31	branches	branch	NOUN	NNS	Number=Plur	25	appos	25:appos	SpaceAfter=No
32	)	)	PUNCT	-RRB-	_	31	punct	31:punct	SpaceAfter=No
33	.	.	PUNCT	.	_	12	punct	12:punct	_
```

```
# sent_id = email-enronsent38_01-0064
# text = He needs more info about these trades (i.e., price info) and would like to talk with you about it.
1	He	he	PRON	PRP	Case=Nom|Gender=Masc|Number=Sing|Person=3|PronType=Prs	2	nsubj	2:nsubj|16:nsubj|18:nsubj:xsubj	_
2	needs	need	VERB	VBZ	Mood=Ind|Number=Sing|Person=3|Tense=Pres|VerbForm=Fin	0	root	0:root	_
3	more	more	ADJ	JJR	Degree=Cmp	4	amod	4:amod	_
4	info	info	NOUN	NN	Number=Sing	2	obj	2:obj	_
5	about	about	ADP	IN	_	7	case	7:case	_
6	these	this	DET	DT	Number=Plur|PronType=Dem	7	det	7:det	_
7	trades	trade	NOUN	NNS	Number=Plur	4	nmod	4:nmod:about	_
8	(	(	PUNCT	-LRB-	_	12	punct	12:punct	SpaceAfter=No
9	i.e.	i.e.	ADJ	FW	_	12	amod	12:amod	SpaceAfter=No
10	,	,	PUNCT	,	_	12	punct	12:punct	_
11	price	price	NOUN	NN	Number=Sing	12	compound	12:compound	_
12	info	info	NOUN	NN	Number=Sing	4	appos	4:appos	SpaceAfter=No
13	)	)	PUNCT	-RRB-	_	12	punct	12:punct	_
14	and	and	CCONJ	CC	_	16	cc	16:cc	_
15	would	would	AUX	MD	VerbForm=Fin	16	aux	16:aux	_
16	like	like	VERB	VB	VerbForm=Inf	2	conj	2:conj:and	_
17	to	to	PART	TO	_	18	mark	18:mark	_
18	talk	talk	VERB	VB	VerbForm=Inf	16	xcomp	16:xcomp	_
19	with	with	ADP	IN	_	20	case	20:case	_
20	you	you	PRON	PRP	Case=Acc|Person=2|PronType=Prs	18	obl	18:obl:with	_
21	about	about	ADP	IN	_	22	case	22:case	_
22	it	it	PRON	PRP	Case=Acc|Gender=Neut|Number=Sing|Person=3|PronType=Prs	18	obl	18:obl:about	SpaceAfter=No
23	.	.	PUNCT	.	_	2	punct	2:punct	_
```

Commit: 2f5eba2

**Comment by amir-zeldes** [comment]
Thanks for reporting the GUM inconsistency, wrong upos was due to incorrect deprel for two instances of i.e. Should be fixed now upstream.

**Comment by muchang** [comment]
Thanks, Amir. BTW, The cases I reported above are in EWT. There is a typo in my original text.

**Comment by amir-zeldes** [comment]
Yes, I saw, but actually there were two upos (but not xpos) errors in GUM due to tree errors, so thanks all the same!

**Comment by nschneid** [comment]
Most fixed in 543e826
